docs/api/address/get-address-single-transaction-with-transfers.schema.json

```diff
@@ -0,0 +1,190 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "description": "GET request that returns information about single account transaction",
+  "title": "AddressTransactionWithTransfers",
+  "type": "object",
+  "required": [
+    "tx_id",
+    "tx_type",
+    "nonce",
+    "fee_rate",
+    "sender_address",
+    "sponsored",
+    "post_condition_mode",
+    "tx_status",
+    "block_hash",
+    "block_height",
+    "burn_block_time",
+    "burn_block_time_iso",
+    "canonical",
+    "is_unanchored",
+    "microblock_hash",
+    "microblock_sequence",
+    "microblock_canonical",
+    "tx_index",
+    "tx_result",
+    "post_conditions",
+    "contract_call",
+    "events",
+    "event_count"
+  ],
+  "properties": {
+    "tx_id": {
+      "type": "string"
+    },
+    "tx_type": {
+      "type": "string"
+    },
+    "nonce": {
+      "type": "integer"
+    },
+    "fee_rate": {
+      "type": "string"
+    },
+    "sender_address": {
+      "type": "string"
+    },
+    "sponsored": {
+      "type": "boolean"
+    },
+    "post_condition_mode": {
+      "type": "string"
+    },
+    "tx_status": {
+      "type": "string"
+    },
+    "block_hash": {
+      "type": "string"
+    },
+    "block_height": {
+      "type": "integer"
+    },
+    "burn_block_time": {
+      "type": "integer"
+    },
+    "burn_block_time_iso": {
+      "type": "string"
+    },
+    "canonical": {
+      "type": "boolean"
+    },
+    "is_unanchored": {
+      "type": "boolean"
+    },
+    "microblock_hash": {
+      "type": "string"
+    },
+    "microblock_sequence": {
+      "type": "integer"
+    },
+    "microblock_canonical": {
+      "type": "boolean"
+    },
+    "tx_index": {
+      "type": "integer"
+    },
+    "tx_result": {
+      "type": "object",
+      "properties": {
+        "hex": {
+          "type": "string"
+        },
+        "repr": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": true
+    },
+    "post_conditions": {
```

Comment:
@MHassanTariq you can use this as a `$ref ` https://github.com/blockstack/stacks-blockchain-api/blob/master/docs/entities/post-conditions/post-condition.schema.json

Reply:
I have used a predefined schema. Updated openapi.yaml with reference to that schema.